amr-wind/wind_energy/ABLWallFunction.cpp

```diff
@@ -328,9 +387,35 @@ void ABLTempWallFunc::wall_model(
                         const amrex::Real vv = vold_arr(i, j, k, 1);
                         const amrex::Real wspd = std::sqrt(uu * uu + vv * vv);
                         const amrex::Real theta2 = told_arr(i, j, k);
-                        tarr(i, j, k - 1) = den(i, j, k) *
-                                            tau.calc_theta(wspd, theta2) /
-                                            alphaT;
+                        amrex::Real ustarthetastar = 0.0;
+                        if (is_multi && is_surf_temp) {
+                            const amrex::Real bh = 9.4;
```

Comment:
are these hardcoded numbers never expected to change?

Reply:
Yes. I can move it to class variables if that's the better way of doing it.